source/_integrations/shelly.markdown

```diff
@@ -43,34 +43,45 @@ Integrate [Shelly devices](https://shelly.cloud) into Home Assistant.
 
 ## Shelly device generations
 
-There are two generations of devices. Both generations are supported by this integration. There are some differences in how devices should be configured and in the naming of entities and devices between generations.
+There are two generations of Shelly devices, and this integration supports both. There are some differences in how devices should be configured and in the naming of entities and devices between generations. Some of the ways to tell the Shelly generations apart are:
 
-## Shelly device configuration (generation 1)
+- In Home Assistant UI, on the Shelly device's page, **Device info** section will state `Hardware: gen1` or `Hardware: gen2`;
+- 2nd generation devices will have a web UI sidebar on the left;
+- 2nd generation devices will have a blue Shelly favicon on their web browser tab.
 
-Generation 1 devices use the `CoIoT` protocol to communicate with the integration. For Shelly firmware 1.10.0 or newer, `CoIoT` must be enabled in the device settings. Navigate to the local IP address of your Shelly device, **Internet & Security** >> **ADVANCED - DEVELOPER SETTINGS** and check the box **Enable CoIoT**.
+> [!NOTE]  
```

Comment:
I'll make these changes throughout the doc. Was initially under the impression that relying on GitHub's native alert box markdown ( https://github.com/orgs/community/discussions/16925 ) instead of styled divs was semantically cleaner, but will switch back to divs.